src/components/DeckBuilderEditorView/index.js

```diff
@@ -113,7 +104,7 @@ class DeckBuilderEditorView extends React.Component {
   }
 
   render() {
-    const matchedDeck = this.isSuggestedDeck()
+    const matchedDeck = isSuggestedDeck(this.props.deck.map(card => card.id))
```

Comment:
```suggestion
    const matchedDeck = isSuggestedDeck(this.props.deck.map(card => card.id))
```
I feel like `isSuggestedDeck` should receive a regular deck and map the cards `id` itself. What do you think?

Reply:
I didn't want to pass too much information but I guess both work - I'll reverse the change to make it clearer in this file